Library/ViewModels/PaymentMethodsViewModelTests.swift

```diff
@@ -303,36 +317,39 @@ internal final class PaymentMethodsViewModelTests: TestCase {
         "Something went wrong and we were unable to remove your payment method, please try again."
       ])
       self.paymentMethods.assertValues(
-        [GraphUserCreditCard.template.nodes, GraphUserCreditCard.template.nodes],
+        [GraphUserCreditCard.template.storedCards, GraphUserCreditCard.template.storedCards],
         "Emits again to reload the tableview after an error occurred"
       )
     }
   }
 
   func testDeletePaymentMethod_SuccessThenError() {
-    guard let card = GraphUserCreditCard.template.nodes.first else {
+    guard let card = GraphUserCreditCard.template.storedCards.first else {
       XCTFail("Card should exist")
       return
     }
 
     let result1 = DeletePaymentMethodEnvelope(storedCards: [card, card])
-
-    let apiService1 = MockService(deletePaymentMethodResult: .success(result1))
+    let response = UserEnvelope<GraphUser>(me: userTemplate)
+    let apiService1 = MockService(
+      deletePaymentMethodResult: .success(result1),
+      fetchGraphUserResult: .success(response)
```

Comment:
Why was fetchGraphUserResult added to all these tests if they weren't here before?

Reply:
Yea, I don't have a great answer here.
I felt comfortable doing this here because these were the view models where we added `fetchGraphUser(withStoredCards: true)` and changed the return type to `Result` instead of a `GraphResponse` in the `MockService`. Short answer is they were set up incorrectly to begin with, so adding these calls fixed pre-existing issues.

Reply:
Gotcha, wasn't sure so thanks for clarifying.